### PR TITLE
Fix DualHits providing 3 judgements instead of two

### PR DESCRIPTION
**Pull request by LumpBloom7**
As simple as it gets, though I did do some minor tweaks to shift input handling and result application responsibility to the DualHitParts, and made the parent do the bare minimum required.

### CODE REVIEW

File: osu.Game.Rulesets.Rush/Objects/DualHit.cs [hunk 1]
```diff
@@ -20,6 +23,10 @@ public override double StartTime
         public readonly DualHitPart Air = new DualHitPart { Lane = LanedHitLane.Air };
         public readonly DualHitPart Ground = new DualHitPart { Lane = LanedHitLane.Ground };
 
+        public override Judgement CreateJudgement() => new RushIgnoreJudgement();
```
Comment (swoolcock) [inline osu.Game.Rulesets.Rush/Objects/DualHit.cs:26]: This removes it from the max combo calculation too, right?

Comment (LumpBloom7) [reply osu.Game.Rulesets.Rush/Objects/DualHit.cs:26]: Yes

**Comment by swoolcock** [comment]
Resolve and I'll merge.

**Comment by swoolcock** [comment]
Looks fine.

**Pull request merged**